teal/legendary/formatter.tl

```diff
@@ -19,6 +19,10 @@ function M.rpad(str: string, len: integer): string
 end
 
 local function col1_str(item: LegendaryItem): string
```

Comment:
If functions can be specific to a mode, shouldn’t we show the modes here?

Reply:
I'm not sure if it makes sense for functions to be specific to a mode... I don't have a strong opinion on it, but I think it's simpler to just remove 'mode' from its type definition, since we pass the visual range regardless and I want it to be closer to a command. Pushed a change (also moved the record to be more consistent).

Reply:
Also, command seems to have a `mode` doc string, but that isn't defined or validated. Is it being used somewhere?

Reply:
Ah, I think the docstring was a copy/paste error. Would you mind fixing that for me while you’re in here? Just remove `---@field description string | nil` from the `LegendaryCommand` type. It’s in the doc comments but not defined on the field and not used. Let’s treat functions the same.

Reply:
Done